### ISSUE TITLE
future.redis:::setAlive(<invalid>) returns NULL on Windows (should be an error)

### ISSUE THREAD

**Issue by HenrikBengtsson** [bwlewis/future.redis#3]
Both of:

```r
future.redis:::setAlive(0,0,"x")}
future.redis:::setAlive(6379L, "127.0.0.1", "x", "auth")
```

return NULL on MS Windows, whereas they give an error on Linux and macOS as expected.

I've patched the package test in PR#1 (https://github.com/bwlewis/future.redis/pull/1/commits/7c7d2aa9bff5bf790266181aa25d152ac02cae73) so that the test passes with OK. I added a `FIXME` comment, so it's not forgotten.

**Comment by bwlewis** [comment]
Was this fixed? I don't see the FIXME comment anymore...

**Comment by HenrikBengtsson** [comment]
> Was this fixed? I don't see the FIXME comment anymore...

Nope. I didn't touch anything under `src/`.  The FIXME:s are in the test script;

https://github.com/bwlewis/future.redis/blob/754778154bc846d7ae589fdfe3e07b5523c89034/tests/catch-all.R#L19-L34
